docs/usage.md

```diff
@@ -0,0 +1,41 @@
+# Usage
+
+## Methodologies
+There are two methodologies for using `angular-fontawesome`. Each has their own benefits and trade-offs but we leave it to you to decide how you'd like to utilize the library.
+
+* [Explicit Reference](./usage/explicit-reference.md)
+* [Icon Library](./usage/icon-library.md)
+
+## Using Additional Styles
+Take advantage of the other FontAwesome icon styles.
+
+* [Brand](./usage/using-other-styles#brand-icons)
```

Comment:
Add `.md` to the path here and two lines below, otherwise link on GitHub does not work.